sql/migrations/20210821150630_world.sql

```diff
@@ -1235,6 +1235,152 @@ INSERT INTO `gameobject` (`guid`, `id`, `map`, `position_x`, `position_y`, `posi
 (12320, 176630, 0, -1680.14, -1562.57, 56.2585, 4.29351, 0, 0, -0.83867, 0.54464, 120, 120, 100, 1, 10);
 UPDATE `gameobject` SET `spawntimesecsmin` = 2, `spawntimesecsmax` = 2 WHERE `id` = 176630;
 
+-- Respawn Frozen Runes

```

Comment:
Shouldn't those be in a pool? I remember hearing they spawn in different places.

Reply:
I'm not sure, my guess would be that they are due to how many there are. I am just unsure of how many should be spawned from the pool.

Reply:
Look in individual parses of full Naxx clear sniffs. This way you can see how many there were in a single instance.